domain/src/main/java/listCategory/Store.java

```diff
@@ -0,0 +1,39 @@
+package listCategory;
+import shop.category.Category;
+import utils.RandomStorePopulator;
+import java.io.*;
+import java.util.List;
+
+public class Store {
+    private List<Category> categoryList;
+
+    public Store() {
+        categoryList = RandomStorePopulator.createListCategory();
```

Comment:
! general comment, why do you invoke populator several times, here, below and in category constructors... from my opinion it should be invoked ones(twice)